roles/mysql/molecule/default/destroy.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Destroy
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: "{{ molecule_no_log }}"
+  tasks:
```

Comment:
Does this play destroy anything? It looks like it dumps the config and leaves the docker container running. Is that on purpose? I'll take a look at some other `destroy.yml` files, but I'll leave this comment here as a reminder.

Reply:
this comes from the `molecule init` command. I've created an issue https://github.com/pulibrary/princeton_ansible/issues/2715